beta/src/pages/learn/writing-markup-with-jsx.md

```diff
@@ -1,60 +1,60 @@
 ---
-title: Writing Markup with JSX
+title: JSX के साथ मार्कअप लिखना
 ---
 
 <Intro>
 
-JSX is a syntax extension for JavaScript that lets you write HTML-like markup inside a JavaScript file. Although there are other ways to write components, most React developers prefer the conciseness of JSX, and most codebases use it.
+JSX जावास्क्रिप्ट के लिए एक सिंटैक्स एक्सटेंशन है जो आपको जावास्क्रिप्ट फ़ाइल के अंदर HTML जैसा मार्कअप लिखने देता है। यद्यपि कौम्पोनॅन्ट्स को लिखने के अन्य तरीके हैं, अधिकांश React डेवलपर्स JSX की संक्षिप्तता को पसंद करते हैं, और अधिकांश कोडबेस इसका उपयोग करते हैं।
 
 </Intro>
 
 <YouWillLearn>
 
-* Why React mixes markup with rendering logic
-* How JSX is different from HTML
-* How to display information with JSX
+- React मार्कअप को रेंडरिंग लॉजिक के साथ क्यों मिलाता है
+- JSX HTML से कैसे अलग है
+- JSX के साथ जानकारी कैसे प्रदर्शित करें
 
 </YouWillLearn>
 
-## JSX: Putting markup into JavaScript {/*jsx-putting-markup-into-javascript*/}
+## JSX: जावास्क्रिप्ट में मार्कअप डालना {/_jsx-putting-markup-into-javascript_/}
```

Comment:
`{/_jsx-putting-markup-into-javascript_/}` => `{/*jsx-putting-markup-into-javascript*/}`

lets not change these, there are used for ref-links